### PR TITLE
MT-7 Topic 2 Home Work

### PR DESCRIPTION
**Pull request by AliakseiDziardziyashenka**
I changed and added files in my Home Work

### CODE REVIEW

**Comment by batonkarton** [comment]
@AliakseiDziardziyashenka let's make one more clean up. 
The folder structure for MentoAD.sln should be as follows:
- MentoAD(repository folder)
  - .github(folder)
  - MentoringHomeWork(folder) - MentoAD.sln should be in this folder (.sln and .csproj files should be in separate folders)
    - HW2 - here you can place your .csproj
      - HW2.0
      - HW2.1

**Comment by batonkarton** [comment]
Now you can merge actual changes from the main branch using GitHub UI:
image

File: .idea/indexLayout.xml
Comment (batonkarton) [inline .idea/indexLayout.xml:1]: Remove .idea folder

File: MentoAD/.idea/.idea.MentoAD/.idea/.gitignore
Comment (batonkarton) [inline MentoAD/.idea/.idea.MentoAD/.idea/.gitignore:1]: Remove .idea folder

File: MentoAD/HW2/HW2.0/HW2.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        string textReaderResult = textReader.ReadToEnd();
+        string[] arrayOfTextResult = textReaderResult.Split(' ');
+        textReader.Dispose();
+        byte[] imageBytes = new byte[arrayOfTextResult.Length - 1];
+        for (int i = 0; i < arrayOfTextResult.Length - 1; i++)
+        {
+
+            byte binary = Convert.ToByte(arrayOfTextResult[i], 2);
+            imageBytes[i] = binary;
+
```
Comment (batonkarton) [inline MentoAD/HW2/HW2.0/HW2.cs]: Remove these whitespaces - let's talk about code style conventions next time - I will prepare a confluence doc about it.

File: MentoAD/HW2/HW2.0/HW2.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        for (int i = 0; i < arrayOfTextResult.Length - 1; i++)
+        {
+
+            byte binary = Convert.ToByte(arrayOfTextResult[i], 2);
+            imageBytes[i] = binary;
+         
+
+        }
+        File.WriteAllBytes(@"/Users/lexadior/Documents/MentoAD/MentoAD/HW2/HW2.0/image100.png", imageBytes);
+
```
Comment (batonkarton) [inline MentoAD/HW2/HW2.0/HW2.cs]: The same.

**Review by batonkarton — CHANGES_REQUESTED** [review]

File: .idea/indexLayout.xml
Comment (batonkarton) [reply .idea/indexLayout.xml:1]: Resolved.

File: MentoAD/.idea/.idea.MentoAD/.idea/.gitignore
Comment (AliakseiDziardziyashenka) [reply MentoAD/.idea/.idea.MentoAD/.idea/.gitignore:1]: Ready for review

**Review by batonkarton — APPROVED** [review]

**Pull request merged**